contrib/docker/odbcinst.ini

```diff
@@ -0,0 +1,15 @@
+[MySQL Driver]
+Driver=/usr/lib/x86_64-linux-gnu/odbc/libmyodbc.so
+UsageCount=1
+Threading=2
+
+[PostgreSQL Driver]
+Driver=/usr/lib/x86_64-linux-gnu/odbc/psqlodbcw.so
+FileUsage=1
+Threading=2
+
+[ODBC Driver 17 for SQL Server]
```

Comment:
Don't hardcode this but use https://github.com/blue-yonder/turbodbc/blob/a4f0c18201a7c766b330c31a029d7297f822f9bd/.travis.yml#L138 to install the correct path as the path changes with each minor version.

Reply:
Well, right now, the path does not change, as I hardcoded the minor version. But I see your point; will add this to `Dockerfile-turbodbc`